react-frontend/src/components/Home/bcLivesteam.js

```diff
@@ -0,0 +1,129 @@
+import React, { useContext } from 'react';
+import dateFormat from 'dateformat';
+import { Col, Row } from 'react-flexbox-grid';
+import { PageContainer } from '../StyleComponents/pageContent';
+import { Heading } from '../StyleComponents/headings';
+import Query from '../Query';
+import { LinkExternalButton } from '../StyleComponents/htmlTags';
+
+import LIVESTREAM_QUERY from '../../queries/digitalBCLivestream/livestreams';
+import { AppConfigContext } from '../../providers/AppConfig';
+const youTubeLogo = require('../../images/icons/yt_logo_rgb_light.png').default;
+
+const BCLivestream = () => {
+  const strapiMediaUrl = useContext(AppConfigContext)['state'][
+    'strapiMediaUrl'
+  ];
+  return (
+    <Query query={LIVESTREAM_QUERY}>
+      {({ data: { digitalLivestreamContact, digitalBcLivestreams } }) => {
+        const imageSource =
+          strapiMediaUrl + digitalBcLivestreams[0]?.EventImage?.url;
+        // if no next event is provided in strapi, this section will not render
+        const now = new Date();
+        const nextEventTime = new Date(digitalBcLivestreams[0]?.BroadcastTime);
+        if (nextEventTime && nextEventTime > now) {
+          return (
+            <div style={{ backgroundColor: '#fff' }}>
+              <PageContainer
+                style={{ backgroundColor: '#fff', paddingTop: '40px' }}
+              >
+                <Row between="xs">
+                  <Col sm={12} md={3}>
```

Comment:
Columns:

- suggest breaking into the three columns at `lg` instead of `md` as this happens:
![Screen Shot 2021-09-22 at 3 17 57 PM](https://user-images.githubusercontent.com/16946297/134433753-af20bfba-a9fd-40db-94e2-14378b32f402.png)

- also need some vertical margin or padding for when the columns stack:
![Screen Shot 2021-09-22 at 4 12 02 PM](https://user-images.githubusercontent.com/16946297/134434100-fd58cfe0-cf60-4ac0-9858-0e6857731298.png)

- and probably a little bit extra horizontal too: 
![Screen Shot 2021-09-22 at 4 18 22 PM](https://user-images.githubusercontent.com/16946297/134434530-260a11a5-3e03-45e5-95ab-41990bb5f125.png)